contentcuration/contentcuration/views.py

```diff
@@ -226,8 +226,8 @@ def _duplicate_node(node, sort_order=1, parent=None):
         fobj_copy.contentnode = new_node
         fobj_copy.save()
 
-    new_node.children = [_duplicate_node(c, parent=None) for c in node.children.all()]
```

Comment:
Creating the nodes with (initially) no parent caused it to generate a new, unique `tree_id` for each. This then wasn't being reset to the `tree_id` of the parent when setting `children`, so as far as mptt was concerned, they were separate trees. This broke functions like `get_family` and `get_descendants`, which filter by `tree_id`.

Confirmed both the problem and the solution through manual testing; now works. @aronasorman @jayoshih: Old databases need a `ContentNode.objects.rebuild()` (I've already run this on production)